tests/end2end/test_insert_mode.py

```diff
@@ -46,7 +46,7 @@ def test_insert_mode(file_name, source, input_text, auto_insert, quteproc):
         quteproc.press_keys(input_text)
     elif source == 'clipboard':
         quteproc.send_cmd(':debug-set-fake-clipboard "{}"'.format(input_text))
-        quteproc.send_cmd(':paste-primary')
+        quteproc.send_cmd(':insert-text {clipboard}')
```

Comment:
I changed it to `{clipboard}` to make it work when the primary selection is not supported. `:paste-primary` fell back on the regular clipboard, but `{primary}` doesn't.